.github/workflows/goreleaser.yaml

```diff
@@ -0,0 +1,32 @@
+name: goreleaser
+
+on:
+  push:
+    # run only against tags
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: git fetch --force --tags
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.19.4'
+          cache: true
+
+      - uses: goreleaser/goreleaser-action@v4
+        with:
+          distribution: goreleaser
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Just leaving a comment as a note for someone to add this secret. 🚀

Reply:
No need, this token is auto-generated per GHA run and inserted by GHA automatically. Its permissions are controlled by the permission block at the top, or its defaults if not set.

Reply:
Oh, sorry yes you're correct, I'm used to doing this on Gitea and needing a token. My bad! 😅